Resources/Prototypes/_RMC14/Entities/Clothing/Belt/lifesaver_bag.yml

```
@@ -107,3 +107,44 @@
     - id: CMHealthAnalyzer
     # TODO RMC: imidazoline pills
     # TODO RMC: alkysine pills
+
+- type: entity
+  parent: CMBeltMedicBag
+  id: RMCBeltMedicBagSPP
+  name: Type 41 pattern medical bag
```

All names are lowercase by default in entity names, one of the exceptions being abbreviations
```suggestion
  name: type 41 pattern medical bag
```